src/test/java/uk/gov/hmcts/reform/sscs/ccd/domain/AdjournCaseDaysOffsetTest.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.hmcts.reform.sscs.ccd.domain;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+class AdjournCaseDaysOffsetTest {
```

Comment:
All these tests have been deleted. Please rebase.

Reply:
rebased and only relevant changes now.